SingleSampleQc.wdl

```diff
@@ -24,11 +24,12 @@ version 1.0
 ## - [VerifyBamID2](https://github.com/Griffan/VerifyBamID)
 
 # Git URL import
-import "https://raw.githubusercontent.com/genome/qc-analysis-pipeline/master/tasks/Qc.wdl" as QC
+#import "https://raw.githubusercontent.com/genome/qc-analysis-pipeline/master/tasks/Qc.wdl" as QC
+import "https://raw.githubusercontent.com/jasonwalker80/qc-analysis-pipeline/wes/tasks/Qc.wdl" as QC
```

Comment:
It's a little bit strange that this change makes sense on the branch but not in the PR!  Does WDL support relative imports like CWL does?

Reply:
WDL supports relative imports but most of the cloud platforms themselves don't.

Reply:
That's a bummer; I guess for the present purpose the PR should be from a branch without 04de35a2f72f65d8dc3603360e0d657fba68530d whereas testing would want to include it.

Reply:
now pointing back to master branch in preparation for the merge.